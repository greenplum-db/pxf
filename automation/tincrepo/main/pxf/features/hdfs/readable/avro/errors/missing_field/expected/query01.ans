-- @description query01 for PXF HDFS Readable Avro with missing field test cases

SELECT * from avro_missing_field;
ERROR:  Avro record has 2 fields but GPDB table has 1 columns.
CONTEXT:  External table avro_missing_field
