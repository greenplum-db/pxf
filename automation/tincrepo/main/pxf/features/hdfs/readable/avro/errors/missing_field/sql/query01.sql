-- @description query01 for PXF HDFS Readable Avro with missing field test cases
-- start_matchsubs
--
-- m/, line \d+ of/
-- s/, line \d+ of .*//
--
-- end_matchsubs

SELECT * from avro_missing_field;
