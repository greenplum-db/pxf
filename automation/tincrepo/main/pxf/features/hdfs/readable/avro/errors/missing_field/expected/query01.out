-- @description query01 for PXF HDFS Readable Avro with missing field test cases
-- start_matchsubs
--
-- m/, line \d+ of/
-- s/, line \d+ of .*//
--
-- end_matchsubs
SELECT * from avro_missing_field;
ERROR:  Avro record has 2 fields but GPDB table has 1 columns.
CONTEXT:  External table avro_missing_field
